src/index.test.ts

```diff
@@ -48,81 +48,93 @@ describe('public convertAxeToSarif API', () => {
+        ${'w3citylights-axe-v4.1.1.reporter-v1.json'}  | ${'w3citylights-axe-v4.1.1.sarif'}
+        ${'w3citylights-axe-v4.1.1.reporter-v2.json'}  | ${'w3citylights-axe-v4.1.1.sarif'}
+        ${'basic-axe-v4.2.0.reporter-v1.json'}         | ${'basic-axe-v4.2.0.sarif'}
+        ${'basic-axe-v4.2.0.reporter-v2.json'}         | ${'basic-axe-v4.2.0.sarif'}
+        ${'w3citylights-axe-v4.2.0.reporter-v1.json'}  | ${'w3citylights-axe-v4.2.0.sarif'}
+        ${'w3citylights-axe-v4.2.0.reporter-v2.json'}  | ${'w3citylights-axe-v4.2.0.sarif'}
+        ${'basic-axe-v4.3.2.reporter-v1.json'}         | ${'basic-axe-v4.3.2.sarif'}
+        ${'basic-axe-v4.3.2.reporter-v2.json'}         | ${'basic-axe-v4.3.2.sarif'}
+        ${'w3citylights-axe-v4.3.2.reporter-v1.json'}  | ${'w3citylights-axe-v4.3.2.sarif'}
+        ${'w3citylights-axe-v4.3.2.reporter-v2.json'}  | ${'w3citylights-axe-v4.3.2.sarif'}
+        ${'basic-axe-v4.4.1.reporter-v1.json'}         | ${'basic-axe-v4.4.1.sarif'}
+        ${'basic-axe-v4.4.1.reporter-v2.json'}         | ${'basic-axe-v4.4.1.sarif'}
+        ${'w3citylights-axe-v4.4.1.reporter-v1.json'}  | ${'w3citylights-axe-v4.4.1.sarif'}
+        ${'w3citylights-axe-v4.4.1.reporter-v2.json'}  | ${'w3citylights-axe-v4.4.1.sarif'}
+        ${'basic-axe-v4.6.3.reporter-v1.json'}         | ${'basic-axe-v4.6.3.sarif'}
+        ${'basic-axe-v4.6.3.reporter-v2.json'}         | ${'basic-axe-v4.6.3.sarif'}
+        ${'w3citylights-axe-v4.6.3.reporter-v1.json'}  | ${'w3citylights-axe-v4.6.3.sarif'}
+        ${'w3citylights-axe-v4.6.3.reporter-v2.json'}  | ${'w3citylights-axe-v4.6.3.sarif'}
+        ${'basic-axe-v4.7.2.reporter-v1.json'}         | ${'basic-axe-v4.7.2.sarif'}
+        ${'basic-axe-v4.7.2.reporter-v2.json'}         | ${'basic-axe-v4.7.2.sarif'}
+        ${'w3citylights-axe-v4.7.2.reporter-v1.json'}  | ${'w3citylights-axe-v4.7.2.sarif'}
+        ${'w3citylights-axe-v4.7.2.reporter-v2.json'}  | ${'w3citylights-axe-v4.7.2.sarif'}
+        ${'basic-axe-v4.8.4.reporter-v1.json'}         | ${'basic-axe-v4.8.4.sarif'}
+        ${'basic-axe-v4.8.4.reporter-v2.json'}         | ${'basic-axe-v4.8.4.sarif'}
+        ${'w3citylights-axe-v4.8.4.reporter-v1.json'}  | ${'w3citylights-axe-v4.8.4.sarif'}
+        ${'w3citylights-axe-v4.8.4.reporter-v2.json'}  | ${'w3citylights-axe-v4.8.4.sarif'}
+        ${'basic-axe-v4.9.1.reporter-v1.json'}         | ${'basic-axe-v4.9.1.sarif'}
+        ${'basic-axe-v4.9.1.reporter-v2.json'}         | ${'basic-axe-v4.9.1.sarif'}
+        ${'w3citylights-axe-v4.9.1.reporter-v1.json'}  | ${'w3citylights-axe-v4.9.1.sarif'}
+        ${'w3citylights-axe-v4.9.1.reporter-v2.json'}  | ${'w3citylights-axe-v4.9.1.sarif'}
+        ${'basic-axe-v4.10.0.reporter-v1.json'}        | ${'basic-axe-v4.10.0.sarif'}
+        ${'basic-axe-v4.10.0.reporter-v2.json'}        | ${'basic-axe-v4.10.0.sarif'}
+        ${'w3citylights-axe-v4.10.0.reporter-v1.json'} | ${'w3citylights-axe-v4.10.0.sarif'}
```

Comment:
do we need the versions that we're not switching to? I'd expect for this update to only add lines with `4.10.2`

Reply:
Updated files for 4.10.2 version